src/securityPolicy.c

```diff
@@ -905,6 +1047,49 @@ security_policy_t policyForSignTxScriptDataHash(const sign_tx_signingmode_t txSi
 	DENY(); // should not be reached
 }
 
+security_policy_t policyForSignTxCollaterals(const sign_tx_signingmode_t txSigningMode)
+{
+	switch (txSigningMode) {
+	case SIGN_TX_SIGNINGMODE_PLUTUS_TX:
+		SHOW();
+		break;
+
+	case SIGN_TX_SIGNINGMODE_ORDINARY_TX:
+	case SIGN_TX_SIGNINGMODE_MULTISIG_TX:
+	case SIGN_TX_SIGNINGMODE_POOL_REGISTRATION_OWNER:
+	case SIGN_TX_SIGNINGMODE_POOL_REGISTRATION_OPERATOR:
+		DENY();
+		break;
+
+	default:
+		ASSERT(false);
+	}
+
+	DENY();
+}
+
+security_policy_t policyForSignTxRequiredSigners(const sign_tx_signingmode_t txSigningMode)
+{
+	//TODO rework when more information is available
```

Comment:
I think we can remove this.

Reply:
Maybe the required signers paths should be validated? (I know it's not necessary, since witness paths are validated, but maybe this is what Marton meant by the comment.)

Reply:
5500f432764c70b1ca703abd95ae6b85af78bac0